central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -599,33 +604,14 @@ func (s *flowStoreImpl) RemoveMatchingFlows(ctx context.Context, keyMatchFn func
 	}
 	defer release()
 
-	// want to do all this as a single transaction
-	tx, err := conn.Begin(ctx)
-	if err != nil {
-		return err
-	}
-
-	// That operation is easy to do in SQL.  If the valueMatchFn is not null then based on the only place
-	// it is set, we are deleting flows orphaned based on elapsed time.
-	if valueMatchFn != nil {
-		// Delete based on orphan time
-		// Do this first because I can do this delete easily via SQL, but until there is a deployment table in PG,
-		// I need to loop over all the IDs in the cluster if keyMatchFn is not null.
-		deleteBefore := time.Now().Add(orphanWindow)
-
-		if _, err := tx.Exec(ctx, deleteOrphanByTimeStmt, s.clusterID, deleteBefore); err != nil {
-			return err
-		}
-	}
-
-	// This operation matches if the either the dest or src deployment no longer exists.
-	// We should be able to do that easily in SQL if we have access to the table with the active deployments.
-	// Then we just have to make sure the functions that do this for Rocks don't change.
-	// For now continue to pull EVERYTHING and compare it to the function which just checks to see if a deployment exists.
-	// If the deployment does not exist, delete the row.
+	// Todo: ROX-9921 Improve this
```

Comment:
Can you re-phrase this with context of keyMatchFn and valueMatchFn? Or you may move it outside the function. That would be helpful to understand the usage of the function also.